### PR TITLE
[apptools] Add sampleapp tests for apptools

### PR DESCRIPTION
**Pull request by yunxliu**
Impacted tests(approved):new 7,update 0,delete 0
Unit test platform:[Android]
Unit test result summary:pass 7,fail 0,block 0

### CODE REVIEW

File: apptools/apptools-sampleapp-tests/README [hunk 1]
```diff
@@ -0,0 +1,55 @@
+## Introduction
+
+This test suite is for apptools-sampleapp-tests
+
+## Common Precondition
+
+1. The node.js must be functional
+2. Install nodeunit: sudo npm install nodeunit -g
+3. Download crosswalk-app-tools to usecase-apptools-tests/tools, then install npm
```
Comment (Honry) [inline apptools/apptools-sampleapp-tests/README]: slip

File: apptools/apptools-sampleapp-tests/README [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+4. The main script is crosswalk-app-tools/src/crosswalk-app. Set environment PATH or invoke with directory
+
+
+## Precondition for AppTools on Android
+
+1. Connect Android devices to your localhost
+2. The Android SDK must be functional
+3. Environment variable configuration:
+  3.1 export ANDROID_HOME=$(dirname $(dirname $(which android)))
+  3.2 export CROSSWALK_APP_TOOLS_CACHE_DIR=/path/to/opt/usecase-apptools-tests/tools
```
Comment (Honry) [inline apptools/apptools-sampleapp-tests/README]: ditto

File: apptools/apptools-sampleapp-tests/README [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+## Precondition for AppTools on iOS
+
+1. Connect iOS devices to your localhost
+2. the Xcode command line tools, and the Xcode iOS SDK 8.1 must be functional
+3. Checkout the iOS backend: cd crosswalk-app-tools/node_modules, then download source code from https://github.com/crosswalk-project/crosswalk-app-tools-ios.git crosswalk-app-tools-backend-ios
+4. Install dependencies: cd crosswalk-app-tools-backend-ios, then npm install, and cd ../..
+
+## Test Steps
+
+1. unzip usecase-apptools-tests<version>.zip -d [testprefix-path]
```
Comment (Honry) [inline apptools/apptools-sampleapp-tests/README]: ditto

File: apptools/apptools-sampleapp-tests/README [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+2. cd [testprefix-path]/opt/usecase-apptools-tests/
```
Comment (Honry) [inline apptools/apptools-sampleapp-tests/README]: ditto

File: apptools/apptools-sampleapp-tests/README [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+3. run test case
+   testkit-lite -f [testprefix-path]/opt/usecase-apptools-tests/tests.xml -M -o [testprefix-path]/opt/usecase-apptools-tests/result.xml --comm androidmobile --deviceid=Medfield3C6DFF2E
```
Comment (Honry) [inline apptools/apptools-sampleapp-tests/README]: ditto

File: apptools/apptools-sampleapp-tests/inst.apk.py [hunk 6]
```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python
```
Comment (Honry) [inline apptools/apptools-sampleapp-tests/inst.apk.py]: This file is unnecessary.

File: apptools/apptools-sampleapp-tests/inst.deb.py [hunk 7]
```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python
```
Comment (Honry) [inline apptools/apptools-sampleapp-tests/inst.deb.py]: ditto

File: apptools/apptools-sampleapp-tests/sampleapp-android/Crosswalk_Apptools_SampleApp_Gallery_Build_Arm.html [hunk 8]
```diff
@@ -0,0 +1,80 @@
...
+    </p>
+    <ol>
+      <li>If com.example.gallery exists in apptools-sampleapp-tests/tools/ directory, remove it</li>
+      <li>Then run below commands under apptools-sampleapp-tests/tools/ directory:
+        <ul>
+          <li>$ crosswalk-app create com.example.gallery</li>
+          <li>$ cd com.example.gallery</li>
+          <li>$ rm -rf app/</li>
+          <li>$ git clone https://github.com/blueimp/Gallery.git</li>
+          <li>Rename folder name Gallery to app</li>
```
Comment (Honry) [inline apptools/apptools-sampleapp-tests/sampleapp-android/Crosswalk_Apptools_SampleApp_Gallery_Build_Arm.html]: What about the manifest.json file in app?

Comment (yunxliu) [inline apptools/apptools-sampleapp-tests/sampleapp-android/Crosswalk_Apptools_SampleApp_Gallery_Build_Arm.html]: The manifest.json file is necessary when pack deb package.

**Comment by Honry** [comment]
LGTM

**Pull request merged**